cookbooks/bcpc-hadoop/recipes/smoke_test_principal.rb

```diff
@@ -62,11 +62,10 @@
 end
 
 chef_vault_secret 'test_user_keytab' do
-  provider ChefVaultCookbook::Provider::ChefVaultSecret
   data_bag 'os'
   raw_data ( lazy {{ 'keytab' => node.run_state['test_user_base64_keytab']}})
   search '*:*'
-  admins "#{ nodes },#{ bootstrap }"
+  admins "#{ nodes },#{ bootstrap },admin"
```

Comment:
this ties us to having running our own chef-server. let's try to keep interacting with the chef-server only through the organization's admin account (which is a normal user with an "admin-level" access for that organization only.

Reply:
At this time there is only this admin account which I have created; I think?